ush/config.sh.RTMA_CONUS_3km

```diff
@@ -249,4 +249,17 @@ if [[ "$MACHINE" == "wcoss_dell_p3" ]]; then
   PTMP="/gpfs/dell2/ptmp/${USER}/ptmp"
   NWGES="/gpfs/dell2/ptmp/${USER}/nwges"
 fi
+# fine tune settings for Jet
+if [[ "$MACHINE" == "jet" ]]; then
```

Comment:
@guoqing-noaa Should we consider using similar settings on WCOSS? I was already considering increasing MAXTRIES to at least 2 for my parallel.

Reply:
If WCOSS has a similar need, I can put these settings outside the if block @MatthewMorris-NOAA

Reply:
Yes, I would suggest placing these settings outside of the if block. Note: "MAXTRIES_RUN_FCST" is already defined on line 180. @guoqing-noaa